backupwordpress.php

```diff
@@ -33,6 +33,10 @@
 
 namespace HM\BackUpWordPress;
 
+if ( ! is_admin() || ( defined( 'DOING_CRON' ) && DOING_CRON ) || ( defined( 'DOING_AJAX' ) && DOING_AJAX ) ) {
```

Comment:
Logic is wrong here, we should `return` is `! DOING_CRON!`, I don't think we'd need to check for `DOING_AJAX` as `admin-ajax.php` already runs in admin context.